### PR TITLE
Fix RMWC links in examples documentation

### PR DESCRIPTION
**Pull request by marvinthepa**

### CODE REVIEW

**Comment by marvinthepa** [comment]
Also fix comcastbusiness.com (which seems offline) to business.comcast.com to fix the build.
Hopefully uber.com works this time 😄 .

**Comment by marvinthepa** [comment]
@tannerlinsley seems like uber is blocking the requests from github. Would it make sense to ignore them for the url checker?

**Comment by tannerlinsley** [comment]
Yeah let’s ignore them

**Comment by gargroh** [comment]
We can also include mdx files for watching broken urls in `Broken Link Checker` to catch broken links like this

File: .github/workflows/links-test.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
           subfolder: /github/workspace/
 
           # A comma-separated list of file types to cover in the URL checks
-          file_types: .md
+          file_types: .md, .mdx
```
Comment (gargroh) [inline .github/workflows/links-test.yml:23]: I think we need to remove `space` after `,` so as to make that valid.

Comment (marvinthepa) [reply .github/workflows/links-test.yml]: It did not complain 😃 .

Anyways, It is easier to change than to test if it works both ways.

Comment (gargroh) [reply .github/workflows/links-test.yml]: Thanks, actually i find in logs for '.mdx' files, it was not showing earlier, now it shows.

Thank you

**Pull request merged**